polygeist/lib/Dialect/Polygeist/Transforms/AffineReduction.cpp

```diff
@@ -346,6 +352,17 @@ class AffineForReductionIter : public OpRewritePattern<AffineForOp> {
       });
     }
 
+    numReductionsDetected += ReductionOps.size();
+
+    DEBUG_WITH_TYPE(
+        REPORT_DEBUG_TYPE, if (!ReductionOps.empty()) {
+          llvm::dbgs()
+              << "AffineReduction: detected " << ReductionOps.size()
+              << " reduction(s) in: "
+              << ForOp->getParentOfType<FunctionOpInterface>().getName()
+              << "\n";
+        });
```

Comment:
minor, for readibility:
```suggestion
    DEBUG_WITH_TYPE(
        REPORT_DEBUG_TYPE, {if (!ReductionOps.empty()) {
          llvm::dbgs()
              << "AffineReduction: detected " << ReductionOps.size()
              << " reduction(s) in: "
              << ForOp->getParentOfType<FunctionOpInterface>().getName()
              << "\n";
        }});
```
the suggestion require clang-format.